vocabularies/Common.xml

```diff
@@ -113,18 +114,18 @@
         </Member>
       </EnumType>
 
-      <Term Name="Timezone" Type="Edm.String" AppliesTo="Property Parameter">
+      <Term Name="Timezone" Type="Edm.String" Nullable="true" AppliesTo="Property Parameter">
```

Comment:
Why do you explicitly set the default value?

Reply:
To express that we mean it to be nullable and haven't just forgotten to say it, as in the other cases.

Also allows to `grep` for potential errors without false positives.